ecs-cluster/keycloak.tf

```diff
@@ -176,22 +176,36 @@ resource "aws_db_subnet_group" "keycloak" {
 }
 
 resource "aws_db_instance" "keycloak" {
-  identifier        = "keycloak"
-  instance_class    = "db.t3.micro"
-  allocated_storage = 5
-  engine            = "postgres"
+  identifier            = "keycloak-1"
+  instance_class        = var.db-instance-type
+  allocated_storage     = 5
+  max_allocated_storage = 20
+  engine                = "postgres"
   # TODO: try serverless?
   # engine = "aurora-postgresql"
   # auto_minor_version_upgrade defaults to True, so this will be auto-upgraded to the  most recent 14.*
-  engine_version         = "14"
+  engine_version    = "14"
+  storage_encrypted = true
```

Comment:
Should we create a specific KMS key for this purpose or will it provision a new one for us?

Reply:
It's automatically created.